src/app/clusters/target-navigator-server/target-navigator-server.cpp

```diff
@@ -39,11 +39,38 @@
  ******************************************************************************/
 
 #include <app/Command.h>
+#include <app/clusters/target-navigator-server/target-navigator-server.h>
+#include <app/common/gen/af-structs.h>
+#include <app/common/gen/cluster-id.h>
+#include <app/common/gen/command-id.h>
+#include <app/common/gen/enums.h>
 #include <app/util/af.h>
 
-bool emberAfTargetNavigatorClusterNavigateTargetCallback(chip::app::Command * commandObj, unsigned char, unsigned char *)
+TargetNavigatorResponse targetNavigatorClusterNavigateTarget(uint8_t target, std::string data);
```

Comment:
Callbacks zapt as followup, please.

Reply:
Added it to the [link](https://github.com/project-chip/connectedhomeip/pull/7906)